app/models/manageiq/providers/embedded_ansible/automation_manager.rb

```diff
@@ -26,6 +26,8 @@ class ManageIQ::Providers::EmbeddedAnsible::AutomationManager < ManageIQ::Provid
   require_nested :Refresher
   require_nested :RefreshWorker
 
+  has_many :configuration_workflows, :dependent => :destroy, :foreign_key => "manager_id", :inverse_of => :manager
```

Comment:
Is this needed?

Reply:
Without this, a bunch specs will fail due to the Inventory collection logic.  I can work on a separate PR just to remove this

Reply:
Right now, the persistor and parser are shared between Embedded and Tower. So not doing this, we will need to add a bunch of conditions to differentiate the 2, or partially separate them. So adding this is actually nicer :-)

So I am fine with Embedded responding to .configuration_workflows, but never refreshing it. (performance wise, this will be recognized as noop in embedded refresh and skipped fast)